execution/scan/subquery.go

```diff
@@ -89,6 +93,15 @@ func (o *subqueryOperator) Next(ctx context.Context) ([]model.StepVector, error)
 		return nil, err
 	}
 
+	scalarArg := 0.0
+	var err error
+	if o.funcExpr.Func.Name == "quantile_over_time" {
+		scalarArg, err = extscanners.UnwrapConstVal(o.funcExpr.Args[0])
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to unwrap scalar argument for quantile_over_time")
```

Comment:
here we also should return something that implies "not implemented" so we trigger fallback into prometheus engine

Reply:
It comes from the unwrap now, but i would prefer marking it here explicitely as not supported instead of reusing the unwrap const val error for that

Reply:
But this an error during execution, right? This won't trigger a fallback anymore. Theoretically this should never happen, because it would be caught before when creating the execution plan.

Reply:
Ah! I missed that and thought it was at creation! True but then again, cannot we unwrap once at construction and remember the value like in matrix selector?